SS14.Launcher/Views/MainWindowTabs/ServerListTabView.xaml

```diff
@@ -3,17 +3,21 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:mainWindowTabs="clr-namespace:SS14.Launcher.ViewModels.MainWindowTabs"
-             xmlns:views="clr-namespace:SS14.Launcher.Views"
-             xmlns:mainWindowTabs1="clr-namespace:SS14.Launcher.Views.MainWindowTabs"
+             xmlns:mainWindowTabsViews="clr-namespace:SS14.Launcher.Views.MainWindowTabs"
              xmlns:loc="clr-namespace:SS14.Launcher.Localization"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
              x:Class="SS14.Launcher.Views.MainWindowTabs.ServerListTabView">
   <Design.DataContext>
     <mainWindowTabs:ServerListTabViewModel />
   </Design.DataContext>
 
-  <DockPanel LastChildFill="True">
+  <UserControl.Styles>
+    <Style Selector=":is(mainWindowTabsViews|ServerList).filtersClosed">
+      <Setter Property="Grid.RowSpan" Value="3"></Setter>
+    </Style>
+  </UserControl.Styles>
```

Comment:
Just confirming here: You've made it so the filter popout isn't overlapping the server list anymore? It's fine, but just want to be clear.

Reply:
Correct – opening the filters will split the window into two panes with their own distinct scrollbars if necessary.
There’s also a grabber in between which can be used to resize the filter portion if the window is big enough.

<img width="912" alt="Screenshot 2025-03-19 at 23 50 27" src="https://github.com/user-attachments/assets/18fe227a-83e6-4186-b290-e6c110d21d3b" />